src/reducers/__tests__/query.spec.jsx

```diff
@@ -711,6 +711,38 @@ describe( 'reducer:query', () => {
       } )
     } )
 
+    describe( 'DATA_LENS_CHANGED actions', () => {
+      it( 'changes the dataLens', () => {
+        const action = {
+          type: actions.DATA_LENS_CHANGED,
+          dataLens: 'foo'
+        }
+        const result = target( {}, action )
+        expect( result ).toEqual( {
+          dataLens: 'foo',
+          enablePer1000: true,
+          mapWarningEnabled: true,
+          queryString: '?dataLens=foo'
+        } )
+      } )
+    } )
+
+    describe( 'DATE_INTERVAL_CHANGED', () => {
+      it( 'changes the dateInterval', () => {
+        const action = {
+          type: actions.DATE_INTERVAL_CHANGED,
+          dateInterval: 'Day'
+        }
+        const result = target( {}, action )
+        expect( result ).toEqual( {
+          dateInterval: 'Day',
+          enablePer1000: true,
+          mapWarningEnabled: true,
```

Comment:
Same.   Maybe we want a Trends reducer?

Reply:
I do have a Trends reducer in the trends work I'm doing.  

These map params comes from function that run after all of the filter actions are run.  I'll add some logic in there so this stuff does not added to the state when it's a trends related action